src/xbot/common/properties/PermanentStorage.java

```diff
@@ -0,0 +1,5 @@
+package xbot.common.properties;
+
+public interface PermanentStorage extends ITableProxy {
```

Comment:
The blank interface is to allow setting a particular implementation in guice land, it just happens to be at this point we don't need anything in addition to ITableProxy right now but if we do need something for storage it can go here.